tests/unit/orders-update.spec.js

```diff
@@ -421,20 +427,28 @@ test(`expandReferences
     .withArgs(mockOrder.syncInfo[0].channel, 'channel', 'channels')
     .returns(channelResult)
 
-  const stateResult = {
+  const lineStateResult = {
     typeId: 'state',
     id: '53 65 6c 77 79 6e',
   }
+  const customLineStateResult = {
```

Comment:
Could also just use 'lineStateResult' since the stubbed function is not part of the test.